homeassistant/components/vilfo/strings.json

```
@@ -11,12 +11,12 @@
       }
     },
     "error": {
-      "cannot_connect": "Failed to connect. Please check the information you provided and try again.",
-      "invalid_auth": "Invalid authentication. Please check the access token and try again.",
-      "unknown": "An unexpected error occurred while setting up the integration."
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": ""
```

As @springstan mentioned, it seems like it was removed by mistake. I noticed that there is a string defined for `error::unknown` in https://github.com/home-assistant/core/blob/dev/homeassistant/strings.json

Something like this should work, I guess:
```suggestion
      "unknown": "[%key:common::config_flow::error::unknown%]"
```

**In all other aspects it looks great!** :+1: :slightly_smiling_face: